mnist-clients.yaml

```diff
@@ -17,9 +17,7 @@ services:
     working_dir: /app/
```

Comment:
I think we should remove this file and only use client.yaml (with --scale argument) for MNIST. Less to maintain.